src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerService.cs

```diff
@@ -1283,6 +1411,109 @@ private void SetIndexesViewInfo(TableDesignerView view)
             view.IndexColumnSpecificationTableOptions.CanInsertRows = false;
         }
 
+        private void SetColumnStoreIndexesViewInfo(TableDesignerView view)
```

Comment:
no sql server version requirement for this feature?

Reply:
All supported version should work, do we need to handle unsupported versions as well?
![image](https://user-images.githubusercontent.com/21186993/188726892-4ecfb20c-051c-43e9-a159-3f4aa5910e4b.png)
https://docs.microsoft.com/en-us/sql/t-sql/statements/create-columnstore-index-transact-sql?view=sql-server-ver16

Reply:
sounds good, just making sure.